documentation/Remove-PnPPlannerBucket.md

```diff
@@ -38,10 +38,10 @@ This cmdlet removes a Planner bucket.
 
 ### Example 1
 ```powershell
-Remove-PnPPlannerBucket -Group "Marketing" -Plan "Conference" -Identity "Preconference Todos"
+Remove-PnPPlannerBucket -Group "Marketing" -Plan "Conference" -Identity "Pre-conference Todo's"
 ```
 
-This removes the "Preconference Todos" bucket from the specified plan.
+This removes the "Pre-conference Todo's" bucket from the specified plan.
```

Comment:
Maybe we can use `Pre-conf checklist` here ? Also,  todos is the correct plural, so maybe we keep it the same ?

Reply:
Todos is fine.